content/pt-br/blog/_posts/2023-03-10-image-registry-change.md

```diff
@@ -0,0 +1,158 @@
+imagem usando o endpoint k8s.gcr.io.
+
+Se você tem o krew instalado, pode instalar o plugin com:
+
+```
+kubectl krew install community-images
+```
+
+E gerar um relatório com:
+
+```
+kubectl community-images
+```
+
+Para métodos alternativos de instalação ou exemplos de saída, veja o repositório [kubernetes-sigs/community-images](https://github.com/kubernetes-sigs/community-image).
+
+**Opção 3**: Se você não tem acesso diretamente ao cluster, ou gerencia muitos clusters - o melhor
+caminho é executar uma busca sobre seus manifestos e charts por _"k8s.gcr.io"_.
+
+**Opção 4**: Se você quer prevenir a execução de imagens a partir do k8s.gcr.io no seu cluster, existem políticas
+para [Gatekeeper](https://open-policy-agent.github.io/gatekeeper-library/website/) e
+[Kyverno](https://kyverno.io/) disponíveis no [repositório de melhores práticas para AWS EKS](https://github.com/aws/aws-eks-best-practices/tree/master/policies/k8s-registry-deprecation) que vão bloquear o pull das imagens. Você
+pode usar essas políticas de terceiros com qualquer cluster Kubernetes.
+
+**Opção 5**: Como a **ÚLTIMA** opção, você pode usar um [webhook de admissão mutating](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#what-are-admission-webhooks) para alterar o
+endereço da imagem dinamicamente. Isso deve ser feito apenas como uma medida paliativa enquanto os
+seus manifestos são atualizados. Você pode encontrar um webhook de admissão mutating e uma política do 
+kyverno (de terceiros) em [k8s-gcr-quickfix](https://github.com/abstractinfrastructure/k8s-gcr-quickfix).
+
+
+## Por que o Kubernetes mudou para um registry de imagem diferente?
+
+O k8s.gcr.io é hospedado em um domínio customizado no [Google Cloud Registry - (GCR)](https://cloud.google.com/container-registry)
+que foi configurado exclusivamente para o projeto do Kubernetes. Isso funcionou desde o nascimento do projeto,
+e nós agradecemos ao Google por prover esses recursos, mas atualmente, existem outros provedores de nuvem e fornecedores
+que gostariam de hospedam imagens para fornecer uma melhor experiência para as pessoas nas suas plataformas. 
+Além do Google ter [renovado o compromisso de doar $3 milhões]((https://www.cncf.io/google-cloud-recommits-3m-to-kubernetes/) para manter a infraestrutura do projeto ano passado, a Amazon Web Services anunciou uma doação correspondente 
+[durante o keynote deles na Kubecon NA 2022 em Detroit](https://youtu.be/PPdimejomWo?t=236). Isso proporcionará
+uma melhor experiência para os usuários (servidores mais pertos = downloads mais rápidos) e vai reduzir 
+a largura de banda de saída e os custos de GCR ao mesmo tempo.
+
+Para mais detalhes sobre essa mudança, leia mais em [registry.k8s.io: faster, cheaper and Generally Available
+(GA)](/blog/2022/11/28/registry-k8s-io-faster-cheaper-ga/).
```

Comment:
Perhaps we could link to the translated version of this blog post and also use the Portuguese title here?